packages/postgres-database/src/simcore_postgres_database/models/users.py

```diff
@@ -100,6 +100,14 @@ class UserStatus(Enum):
         nullable=True,  # since 2FA can be configured optional
         doc="Confirmed user phone used e.g. to send a code for a two-factor-authentication",
     ),
+    sa.Column(
+        "two_factor_enabled",
+        sa.Boolean,
+        server_default=sa.sql.expression.true(),
+        nullable=False,
+        doc="Wheter 2FA is enabled at login by this user."
+        "NOTE that this is checked ONLY if application activates 2FA",
```

Comment:
Question: why is this on the user? I thought this was linked to the product? the users table is not linked to a product right?
we have users, then users_to_groups, then groups_extra_properties.
so that means if a user defines 2fa authentication in this case, it will be enabled on all products right?

Reply:
I tried to explain the logic in the description. 

Yes, right now the 2FA is set at the app or product level. 
The purpose of this PR is to set is as a property of the user because of the limitations that we have now it is in some cases too restrictive and in some cases too insecure. 

Regarding your suggestion to add it as property of a group, I would not agree. 2FA is an option that **the user** typically want to enable or disable and not any other group. In our case it is imposed by requirements but IMO this will change pretty soon.

I details explain above.